MarkLogic New Integration (#6524)
DataDog/integrations-core: marklogic/tests/README.md

# Test environment for MarkLogic

> This integration can be tested using the Docker Compose environment.

Reviewer: ```suggestion
Test this integration using the Docker Compose environment.
```

> In order to have the MarkLogic docker image, you need to `docker login` and get the docker image here https://hub.docker.com/_/marklogic.

Reviewer: ```suggestion
To get the MarkLogic Docker image, run `docker login` and download the Docker image here https://hub.docker.com/_/marklogic.
```
is download the right word?

## Cluster mode

> The cluster environment is not working on CI yet (TODO).

Reviewer: okay with this going out like this?
